index.bs

```diff
@@ -542,7 +643,7 @@ When this method is invoked, the user agent MUST execute the following algorithm
         execute a platform-specific procedure to determine which, if any, credentials listed in {{AssertionOptions/allowList}}
         might be present on this authenticator, and set |credentialList| to this filtered list. If no such filtering is
         possible, set |credentialList| to an empty list.
```

Comment:
Should that be [=list=]? Or were you planning to make getAssertion more precise along similar lines in a new PR?